_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_labels.scss

```diff
@@ -0,0 +1,8 @@
+.pf-c-label {
+  &.rhd-u-label-large {
```

Comment:
I intended to use this on the activeFilters labels to make them larger;  I did not see a modifier to make labels small/large/etc ([link](https://www.patternfly.org/v4/documentation/react/components/label)) 

If there code changes from the review,  I will add this to those active filters.  Please let me know if the utility (u) class name follows standards.